style.css

```diff
@@ -0,0 +1,80 @@
+* {
+  font-family: Helvetica, sans-serif;
+}
+
+header,
+footer {
+  background: #333;
+  height: 100px;
+}
+
+nav {
+  /* border: 3px dashed orange; */
+  width: 960px;
+  margin: 0px auto;
+}
+
+header img {
+  float: left;
+  margin-top: 15px;
+  margin-right: 60px;
+}
+
+nav li {
+  display: inline-block;
+  margin-top: 40px;
```

Comment:
Instead of using a margin-top, you can use `line-height: 100px;` for vertical alignment of inline elements.